### ISSUE TITLE
Display uk_happy in city's Supported units

### ISSUE THREAD

**Issue by ec429** [longturn/freeciv21#934]
**Is your feature request related to a problem? Please describe.**
Currently, the "Supported units" box in the city UI displays each unit's upkeep in food/shields/gold, but not happiness, making it harder to see where military unhappiness is coming from.

**Describe the solution you'd like**
Add the happiness upkeep to the box, both in the mouseover text and graphically with the `upkeep.unhappy#` icons.

**Describe alternatives you've considered**
???

**Additional context**

**Comment by jwrober** [comment]
Is this resolved in PR #1213 ?

**Comment by ec429** [comment]
Not really; afaict that only displays the _total_ `uk_happy` for the city — it doesn't indicate which units it's coming from.

**Comment by jwrober** [comment]
May be related to #1859 and #1873
